components/new_post/post_component.tsx

```diff
@@ -0,0 +1,498 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {MouseEvent, useCallback, useEffect, useRef, useState} from 'react';
+import {FormattedMessage} from 'react-intl';
+import classNames from 'classnames';
+
+import {Posts} from 'mattermost-redux/constants/index';
+import {
+    isMeMessage as checkIsMeMessage,
+    isPostPendingOrFailed,
+} from 'mattermost-redux/utils/post_utils';
+
+import Constants, {A11yCustomEventTypes, AppEvents, Locations} from 'utils/constants';
+import * as PostUtils from 'utils/post_utils';
```

Comment:
may be selectively import ?

Reply:
I did it for the ones that we import only one function but here we import 6, is that something we wanna create a convention for?

Reply:
i dont think there is any convention. Should'nt we import only what we need.